ansible/roles/linux-data-volumes/files/data-volumes-playbook.yml

```diff
@@ -13,3 +13,7 @@
       loop_control:
         loop_var: data_volume
         label: "{{ data_volume.mountpoint }}"
+      when: data_volumes.configure_nvme is undefined
+
+    - include_tasks: /etc/ansible-init/includes/data-volumes-configure-nvme.yml
+      when: data_volumes.configure_nvme
```

Comment:
Check for "lvm_disk_nvme" in openstack_metadata.

Probably this should be a separate role.